data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -2059,6 +2163,136 @@ spec:
                 - ovirt_cluster_id
                 - ovirt_storage_domain_id
                 type: object
+              powervs:
+                description: PowerVS is the configuration used when installing on
+                  Power VS.
+                properties:
+                  APIKey:
+                    description: APIKey is the API key for the user's IBM Cloud account.
+                    type: string
+                  SSHKeyName:
+                    description: SSHKeyName is the name of an SSH key stored in the
+                      Service Instance.
+                    type: string
+                  bootstrapOSImage:
+                    description: BootstrapOSImage is a pre-created boot image to override
+                      the default OS image for the bootstrap node.
+                    type: string
+                  clusterOSImage:
+                    description: "ClusterOSImage is a pre-created Power VS boot image
```

Comment:
once again these are referring to the bootimages - which is not what is supplied through the rhcos json. these would be internally created through the ova images?

Reply:
lol, once again -- bootimages. and yes :)